Tasks/Shurko/Task1/Task1.py

```diff
@@ -0,0 +1,8 @@
+
+
+d=20000   # initial deposit
+t=5       # deposit term in year
+n=15      # percent per year
+p=12      #capitalization period in monthe
```

Comment:
would be better to use the decimal data type here to prevent additional calculation error, -1 point for the unwise choice